app/scripts/components/common/page-footer/footer.test.tsx

```diff
@@ -46,14 +44,17 @@ describe('PageFooter', () => {
   });
   test('renders the PageFooter', () => {
     render(
-      <PageFooter
-        mainNavItems={mockMainNavItems}
-        subNavItems={mockSubNavItems}
-        hideFooter={hideFooter}
-        logoSvg={<NasaLogoColor />}
-        linkProperties={mockLinkProperties}
-        footerSettings={defaultFooterSetting}
-      />
+      <BrowserRouter basename=''>
```

Comment:
We should mock the BrowserRouter, either for this test only, or in `jest.setup.ts`, to isolate component behavior, reduce rendering complexity and eventually have faster tests. 
Alternatively, you might consider using [MemoryRouter](https://reactrouter.com/6.29.0/router-components/memory-router) instead of mocking, as it's specifically designed for testing.